src/Helpers/TableToolbarHelper.js

```diff
@@ -39,12 +44,19 @@ export const buildActiveFilters = (currentFilters, filterRulevalues = []) => {
     };
 
     const filterChips = Object.keys(FILTERS).reduce((array, key) => {
-        if (key !== 'show_all' && Object.keys(currentFilters).includes(key) && !['', undefined].includes(currentFilters[key])) {
-
+        if (key === 'security_rule'
+            && (!['', undefined].includes(currentFilters.rule_presence) || !['', undefined].includes(currentFilters.rule_key))) {
+            const multiValue = concatArrays(
+                currentFilters.rule_presence && currentFilters.rule_presence.split(','),
+                currentFilters.rule_key && currentFilters.rule_key.split(','));
```

Comment:
why do you need to concat `rule_presence` and `rule_key` since you're blocking the values `undefined` and `''` in the if statement? ex `!['', undefined].includes(currentFilters.rule_presence)`

Also, `!['', undefined].includes(VALUE)` can be a method to be re-used

Reply:
I am blocking rule_presence and rule_key BOTH being nullish but in the concat I am testing if EITHER of them are nullish.
So if rule_presence has value, but rule_key does not, it passes the condition in if and fails condition inside concat.